test/comm_config.yaml

```diff
@@ -24,21 +24,21 @@ communications:
     enabled: true
     channel: 'cloud-alerts'
     token: 'ABCDEFG'
-    notiftype: short                           # Change notification type short/long you want to receive. notiftype is optional and Default notification type is short (if not specified)
+    notiftype: short                            # Change notification type short/long you want to receive. notiftype is optional and Default notification type is short (if not specified)
   
   # Settings for Mattermost
   mattermost:
     enabled: false
     url: 'MATTERMOST_SERVER_URL'                # URL where Mattermost is running. e.g https://example.com:9243
-    token: 'MATTERMOST_TOKEN'			# Personal Access token generated by BotKube user
-    team: 'MATTERMOST_TEAM'			# Mattermost Team to configure with BotKube 
-    channel: 'MATTERMOST_CHANNEL'		# Mattermost Channel for receiving BotKube alerts 
-    notiftype: short                           # Change notification type short/long you want to receive. notiftype is optional and Default notification type is short (if not specified)
+    token: 'MATTERMOST_TOKEN'			              # Personal Access token generated by BotKube user
+    team: 'MATTERMOST_TEAM'			                # Mattermost Team to configure with BotKube 
+    channel: 'MATTERMOST_CHANNEL'	              # Mattermost Channel for receiving BotKube alerts 
+    notiftype: short                            # Change notification type short/long you want to receive. notiftype is optional and Default notification type is short (if not specified)
```

Comment:
Please fix the indentation. I think you are using mix of tabs and spaces. Let's stick with spaces only

Reply:
my editor was dint show me the difference, however fixed these some how